pennylane/tape/tapes/cv_param_shift.py

```diff
@@ -236,18 +236,30 @@ def parameter_shift_first_order(
         p_idx = self._par_info[t_idx]["p_idx"]
 
         recipe = op.grad_recipe[p_idx]
-        c, s = (0.5, np.pi / 2) if recipe is None else recipe
+
+        # Default values
+        multiplier = 0.5
+        a = 1
+        shift = np.pi / 2
+
+        # We set the default recipe to as follows:
+        # ∂f(x) = c1*f(a1*x+s1) + c2*f(a2*x+s2)
+        default_param_shift = [[multiplier, a, shift], [-multiplier, a, -shift]]
+        param_shift = default_param_shift if recipe is None else recipe

```

Comment:
With the changes in my previous comment, this could be replaced with simply `op.get_parameter_shift()` (unless I'm missing something?)

Reply:
Opened #924